content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md

```diff
@@ -0,0 +1,33 @@
+Does your Cluster Agent restart all the time on AKS?
+
+It might be because you have deployed it with our helm chart and the readiness probe is killing it.
+On AKS, the default RBAC does not allow access to `/healthz` for unauthenticated users.
+The following RBAC fixes that:
+
+```
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:healthz
+rules:
+- nonResourceURLs:
+  - /healthz
+  verbs:
+  - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:healthz
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:healthz
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: system:authenticated
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: system:unauthenticated
```

Comment:
Do we need the `system:unauthenticated` ?

Reply:
@CharlyF who should we ping for the answer to this?

Reply:
@xvello any insights?

Reply:
This is due to the livenessprobe [defined here when custom metrics are enabled](https://github.com/helm/charts/blob/5a1ab776a91aa9119939f228009522d5e811bba7/stable/datadog/templates/cluster-agent-deployment.yaml#L93-L99)

The kubelet curls `dca-pod:443/healthz` with no authentication headers, so the request will fall into the `system:unauthenticated` group.

Reply:
Thanks! @cswatt can we merge this one now that the modifications are done?
Let me know if I need to do anything else.

Thanks!